fastlane/Fastfile

```diff
@@ -9,7 +9,9 @@ APP_SPECIFIC_VALUES = {
     package_name: 'org.wordpress.android',
     bundle_name_prefix: 'wpandroid',
     screenshots_test_class: 'org.wordpress.android.ui.screenshots.WPScreenshotTest',
-    screenshot_config_file: 'wordpress-config.json'
+    screenshot_config_file: 'wordpress-config.json',
+    test_targets: 'notPackage org.wordpress.android.ui.screenshots'
+    
```

Comment:
🔍 Nitpick: Given those constants are declared in the global space, I'd suggest to name that key something more like `:uitest_targets` or `instrumented_test_target`. That way if later we need a similar per-app key/constant but for _Unit_ Tests, we'd not risk confusion between the two.

Reply:
Yup that makes sense, updated in https://github.com/wordpress-mobile/WordPress-Android/pull/17950/commits/26e76361a98898d2e082748fe96e0e8f60ec8366